react/components/Slider/index.js

```diff
@@ -93,9 +93,10 @@ export default class Slider extends Component {
     window.removeEventListener('resize', this.updateLayout)
 
     if (this.cancelDragEvent_) {
```

Comment:
the condition says that if `this.cancelDragEvent_` exists then run `this.cancelDragEvent _ ()` and `this.cancelDragEvent_ = undefined` and then return.

the way you did it, it's the other way around!

The correct should be:
```suggestion
    if (!this.cancelDragEvent_) {
```

Reply:
My bad didn't notice it :) fixed it, thanks!